integration_test/tests/connection_tests.rs

```diff
@@ -22,6 +26,49 @@ mod integration {
         let conn_str = crate::common::generate_default_connection_str();
         let _ = connect_with_conn_string(env_handle.unwrap(), conn_str).unwrap();
     }
+    #[test]
+    fn test_connection_with_log_level() {
+        let env_handle = allocate_env();
+        let conn_str = format!(
+            "{};{}",
+            crate::common::generate_default_connection_str(),
+            "LOGLEVEL=DEBUG"
+        );
+        let _ = connect_with_conn_string(env_handle.unwrap(), conn_str).unwrap();
+    }
+
+    #[test]
+    fn test_logs_exist() {
+        use std::{fs, path::Path};
+        let mut buffer = [0u16; 1024];
+        unsafe {
+            SQLGetPrivateProfileStringW(
+                to_widechar_ptr(DRIVER_NAME).0,
+                to_widechar_ptr("Driver").0,
+                to_widechar_ptr("").0,
+                buffer.as_mut_ptr(),
+                buffer.len() as i32,
+                to_widechar_ptr("odbcinst.ini").0,
+            )
+        };
+
+        let driver_path = unsafe { cstr::parse_attribute_string_w(buffer.as_mut_ptr()) };
+
+        let log_file_path = Path::new(&driver_path)
+            .parent()
+            .unwrap()
+            .parent()
+            .unwrap()
+            .join("logs/mongo_odbc.log")
```

Comment:
The advantage of join is it uses the correct file separator, so I think we should join `logs` and `mongo_odbc.log` even though Windows pretty much doesn't care \ vs / anymore.

I know I'm using `/` explicitly in the postinstall script for macos, but it's specifically not to be used on windows (lol)

Reply:
In testing it was able to handle the forward slash, but I've now changed it everywhere to use `Path` to make doubly sure.